app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -77,7 +76,7 @@ import org.mozilla.fenix.wallpapers.WallpaperManager
 fun WallpaperSettings(
     wallpapers: List<Wallpaper>,
     defaultWallpaper: Wallpaper,
-    loadWallpaperResource: (Wallpaper) -> Bitmap?,
+    loadWallpaperResource: suspend (Wallpaper) -> Bitmap?,
```

Comment:
didn't even think that lambdas could be `suspends`. neat

Reply:
Totally! `suspend` is what `@Composable` is modeled after, I believe so the two are basically interchangeable (in terms of how they affect what they modify)